test/Aggregate/AggregateHydratorFunctionalTest.php

```diff
@@ -7,6 +7,8 @@
  * @license   http://framework.zend.com/license/new-bsd New BSD License
```

Comment:
License docblocks needs update, as mentioned before: `@see` -> `@link`, https links and correct link to license.

Everywhere in tests.